components/lib/push-amazon/build.gradle

```diff
@@ -24,7 +24,11 @@ android {
 }
 
 dependencies {
+    compileOnly files('libs/amazon-device-messaging-1.0.1.jar')
```

Comment:
Since `compileOnly` is deprecated for `implementation` and `api`, do neither of those work?

Reply:
`compileOnly` is to replace deprecated `provided`; different from runtime available `implementation` or `api`

Reply:
https://blog.gradle.org/introducing-compile-only-dependencies

Reply:
Ah my bad, I was still thinking about `compile`. :)